src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.ColorSupplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int NUMBER_OF_FIGURES = 20;

```

Comment:
why this constant called NUMBER_OF_FIGURES? And dont forget about access modifier